README.md

```diff
@@ -128,15 +102,7 @@ LibRaw is a library that processes RAW files from digital cameras. It handles im
 	```
 	
 * macOS
-	
-	Install homebrew if not already installed
-	
-	```sh
-	/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-	```
-	
-	Install libraw
-	
```

Comment:
Removing repeating instruction to declutter. Kept the first one. IF people following to top to bottom, they would installed homebrew already, or they can always google